### PR TITLE
Using timestamps in static names for Bigquery tests

### PR DESCRIPTION
**Pull request by frankyn**
There's an issue where Travis and Circle fail each others tests, because they're using the same static name to test with. This PR will add time to generate unique names.

@remi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Ruby docs samples
 
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/ruby-docs-samples.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/ruby-docs-samples)
+[![CircleCI](https://circleci.com/gh/GoogleCloudPlatform/ruby-docs-samples/tree/master.svg?style=svg)](https://circleci.com/gh/GoogleCloudPlatform/ruby-docs-samples/tree/master)
```
Comment (remi) [inline README.md]: How come this is part of this PR?

File: bigquery/spec/bigquery_sample_spec.rb [hunk 2]
```diff
@@ -28,35 +28,40 @@
     @bucket     = @storage.bucket ENV["GOOGLE_CLOUD_STORAGE_BUCKET"]
     @tempfiles  = []
 
-    # Examples assume that newly created test_dataset and test_table exist
-    delete_test_dataset!
+    file_time     = Time.now.to_i
```
Comment (remi) [inline bigquery/spec/bigquery_sample_spec.rb]: What does "file time" mean?

What about calling this `timestamp` or `current_time`?

Comment (frankyn) [inline bigquery/spec/bigquery_sample_spec.rb]: Agreed, this can be named better!

File: README.md (hunk 1 above)
Comment (frankyn) [inline README.md]: Oops?? I merged in Master to make sure they were on the same commit. I'll fix that.

**Pull request merged**